app/ydoc-shared/src/languageServerTypes/suggestions.ts

```diff
@@ -13,9 +13,9 @@ export interface SuggestionEntryArgument {
   /** Indicates whether the argument has default value. */
   hasDefault: boolean
   /** Optional default value. */
-  defaultValue?: string
+  defaultValue?: string | null
   /** Optional list of possible values that this argument takes. */
-  tagValues?: string[]
+  tagValues?: string[] | null
```

Comment:
These structures reflect their [protocol definitions](https://github.com/enso-org/enso/blob/d87484b9b2eb2395499a19b59be41866c223b781/docs/language-server/protocol-language-server.md#suggestionentryargument) and I would consider leaving them this way. If someone would in the future send SuggestionEntryArgument to LS with nulls, those will be rejected by engine. On the other hand, there's currently no such endpoint in LS, so I leave this decision to you.